.prettierrc.yml

```diff
@@ -1,3 +1,3 @@
-printWidth: 120
+printWidth: 100
```

Comment:
Any reason for this?
Personally I have always used 120 that is why this was at 120. 
Is this just a personal preference?

Reply:
Yeah no reason other than I think this the default in Prettier and what I had in the Octane repo. Happy to change back.